jtabba/CDW-1
# Fixed spacing between header, body, and footer

## Type of change

[] New feature
<br />
[x] Bug fix
<br />
[] Documentation
<br />
[] Code refactor/tech debt
<br />
[] Infrastructure


## Description

Header, body, and footer now automatically stay apart (header at the top, footer at the bottom.)
Also cleaned up some css on the footer that caused overflow.


## Ticket link

[CDW-52](https://linear.app/cdw/issue/CDW-52/footer-should-sit-at-bottom-of-page)

## Part of a back end implementation?
[x] No
<br />
[] [Yes]()  


## How to test


1. Checkout to this branch
2. From /frontend, npm start
3. Navigate through pages
4. Check that footer stays at the bottom

## Screenshots

![image](https://github.com/jtabba/CDW-1/assets/12403004/bdc0e9ba-9f9b-4daf-a4dd-aef5cab41a57)
![image](https://github.com/jtabba/CDW-1/assets/12403004/0cadf2e7-4464-4673-a0d9-241af2c0cccc)
